test/integration/role.js

```diff
@@ -0,0 +1,62 @@
+/* global expect, chai, agent */
+
+const helpers = require('./helpers');
+
+/*
+ * The /roles API endpoint
+ *
+ * This test suite implements full CRUD on the /projects HTTP API endpoint.
+ */
+describe('(/roles) The roles API endpoint', () => {
+  // project we will add during this test suite.
+  const roles = {
+    label : 'Receptionniste',
+    project_id : 1,
+  };
+  const adminUuid = '5b7dd0d6-9273-4955-a703-126fbd504b61';
+  const roleAdmin = {
+    label : 'Administrator',
+  };
+  const userRole = {
+    user_id : 2,
+    role_uuids : adminUuid,
+  };
+
+  it('GET /roles returns a list of roles', () => {
+    return agent.get('/roles?project_id='.concat(roles.project_id))
+      .then((res) => {
+        expect(res).to.have.status(200);
+        expect(res.body).to.not.be.empty;
+      })
+      .catch(helpers.handler);
+  });
+
+  it('POST /roles add a new role', () => {
+    return agent.post('/roles')
+      .send(roles)
+      .then((res) => {
+        expect(res).to.have.status(200);
```

Comment:
Normally, POST should send back 201 if it was created.  Check out the [w3 spec](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html) on it for detailed information, or our own [API Guidelines](https://github.com/IMA-WorldHealth/bhima-2.X/wiki/Design:-Server-CRUD-APIs).